src/Package.appxmanifest

```diff
@@ -47,20 +47,6 @@
             </com:ExeServer>
           </com:ComServer>
         </com:Extension>
-        <uap3:Extension Category="windows.appExtension">
-          <uap3:AppExtension Name="com.microsoft.devhome" Id="PG-SP-ID1" PublicFolder="Public" DisplayName="ms-resource:AppDisplayNameDev" Description="ms-resource:AppDescription">
-            <uap3:Properties>
-              <DevHomeProvider>
-                <Activation>
-                  <CreateInstance ClassId="426A52D6-8007-4894-A946-CF80F39507F1" />

```

Comment:
This Guid is indeed in the core widget extension definition: 

[Guid("426A52D6-8007-4894-A946-CF80F39507F1")]
[ComDefaultInterface(typeof(IExtension))]
public sealed class CoreExtension : IExtension

Reply:
Oh ya... fixed :(